lib/tasks/manage.rake

```diff
@@ -2,7 +2,7 @@ require 'csv'
 
 namespace :manage do
   def change_email_address(old_email_address:, new_email_address:)
-    subscriber = Subscriber.find_by("LOWER(address) = ?", old_email_address.downcase)
+    subscriber = Subscriber.find_by_address(old_email_address)
```

Comment:
is it worth using `find_by!` and relying on the `RecordNotFound` error rather than rolling our own on the next line?

Reply:
Could well do. I assumed it was for a clearer error message, but didn't really ponder how useful/superfluous it is.

Reply:
cool. no matter at this point anyway. the code works.